root/statistics/stats.js

```diff
@@ -1,4 +1,5 @@
 /*
+ *
```

Comment:
Intentional?

In this file, please do check the translations for labels such as "l('Failed edits (voted down)')". They're not getting translated, and neither do they if I change that to l_statistics (which seems like the right call).